sites/default/LBF/LBFPHQ9.plugin.php

```diff
@@ -0,0 +1,104 @@
+<?php
+// Copyright (C) 2017 Sherwin Gaddis <sherwin@openmedpractice.com>
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+
+// This provides enhancement functions for the referral (REF) form.
+// It is invoked by interface/patient_file/transaction/add_transaction.php.
+
+// The purpose of this function is to create JavaScript for the <head>
+// section of the page.  This in turn defines desired javaScript
+// functions.
+//
+// @author Sherwin Gaddis <sherwin@openmedpractice.com>
+// @edited by Joyce Boyd <jboyd13@masonlive.gmu.edu>
+//   added code to update Referral Answer 
+//
+
+
+
+function LBFPHQ9_javascript() {
```

Comment:
You could instead make this LBFPHQ9_javascript_onload() and thus not need $(document).ready, as the function would run once after the page is loaded.